sklearn/calibration.py

```diff
@@ -176,23 +176,10 @@ def fit(self, X, y, sample_weight=None):
         self : object
             Returns an instance of self.
         """
-        X, y = self._validate_data(X, y, accept_sparse=['csc', 'csr', 'coo'],
-                                   force_all_finite=False, allow_nd=True)
         X, y = indexable(X, y)
         le = LabelBinarizer().fit(y)
         self.classes_ = le.classes_
```

Comment:
I am a bit curious about these lines here. In case, that the model has been prefitted, it would be best to use `prefitted_model.classes_` instead? Meaning that we have this part only when the model is not prefitted.